pkg/constants/constants.go

```diff
@@ -147,6 +147,9 @@ const (
 
 	// OSMConfigMap is the name of the OSM ConfigMap
 	OSMConfigMap = "osm-config"
```

Comment:
remove.

Reply:
This cannot be removed as its being referenced in the validating webhook code, once that code is removed we can remove this